src/main/java/supersymmetry/common/metatileentities/multi/electric/MetaTileEntityEvaporationPool.java

```diff
@@ -0,0 +1,104 @@
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABBBBBBBBBBBBBBBBBBBBBBBBBBBBBBA", " BBBBBBBBBBBBBBBBBBBBBBBBBBBBBB ")
+                .aisle("AAAAAAAAAAAAAAAASAAAAAAAAAAAAAAA", "                                ")
+                .where('S', selfPredicate())
+                .where('A', states(MetaBlocks.STONE_BLOCKS.get(StoneVariantBlock.StoneVariant.SMOOTH).getState(StoneVariantBlock.StoneType.CONCRETE_LIGHT))
+                        .or(autoAbilities(true, true, true, true, true, true, false)))
```

Comment:
You can just do `autoAbilities()` here - the muffler mechanics are controlled by another method which needs overriding to enable.